upgrade-notes/storefront_20240906_105852.md

```diff
@@ -0,0 +1,3 @@
+#### add usersnap widget ([#3408](https://github.com/shopsys/shopsys/pull/3408))
+
+-   see #project-base-diff to update your project
```

Comment:
maybe mention a new env variable that should be set during deployment to use usersnap?

Reply:
added more descriptive upgrade notes in 7dea2cdae9